<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-1J0QRU3" ignored="0" name="用户管理系统测试用例" tests="7" failures="4" timestamp="14 二月 2019 09:21:19 GMT" time="14.474" errors="0">
  <testcase name="loginTrue" time="2.456" classname="com.course.cases.LoginTest"/>
  <testcase name="loginFalse" time="1.079" classname="com.course.cases.LoginTest"/>
  <testcase name="addUser" time="3.108" classname="com.course.cases.AddUserTest">
    <failure type="org.apache.ibatis.exceptions.PersistenceException" message="
### Error querying database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for addUser
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for addUser">
      <![CDATA[org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for addUser
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for addUser
at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:30)
at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:150)
at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:141)
at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:77)
at com.course.cases.AddUserTest.addUser(AddUserTest.java:36)
Caused by: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for addUser
at org.apache.ibatis.session.Configuration$StrictMap.get(Configuration.java:888)
at org.apache.ibatis.session.Configuration.getMappedStatement(Configuration.java:721)
at org.apache.ibatis.session.Configuration.getMappedStatement(Configuration.java:714)
at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:147)
... 28 more
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- addUser -->
  <testcase name="getUserInfo" time="3.012" classname="com.course.cases.GetUserInfoTest">
    <failure type="org.apache.ibatis.exceptions.PersistenceException" message="
### Error querying database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 111
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 111">
      <![CDATA[org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 111
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 111
at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:30)
at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:150)
at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:141)
at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:77)
at com.course.cases.GetUserInfoTest.getUserInfo(GetUserInfoTest.java:35)
Caused by: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 111
at org.apache.ibatis.session.Configuration$StrictMap.get(Configuration.java:888)
at org.apache.ibatis.session.Configuration.getMappedStatement(Configuration.java:721)
at org.apache.ibatis.session.Configuration.getMappedStatement(Configuration.java:714)
at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:147)
... 28 more
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- getUserInfo -->
  <testcase name="updateuserinfo" time="0.845" classname="com.course.cases.UpdateUserInfoTest"/>
  <testcase name="deleteuser" time="3.06" classname="com.course.cases.UpdateUserInfoTest">
    <failure type="org.apache.ibatis.exceptions.PersistenceException" message="
### Error querying database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 删除成功
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 删除成功">
      <![CDATA[org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 删除成功
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 删除成功
at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:30)
at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:150)
at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:141)
at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:77)
at com.course.cases.UpdateUserInfoTest.deleteuser(UpdateUserInfoTest.java:44)
Caused by: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 删除成功
at org.apache.ibatis.session.Configuration$StrictMap.get(Configuration.java:888)
at org.apache.ibatis.session.Configuration.getMappedStatement(Configuration.java:721)
at org.apache.ibatis.session.Configuration.getMappedStatement(Configuration.java:714)
at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:147)
... 28 more
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- deleteuser -->
  <testcase name="getuserlist" time="0.825" classname="com.course.cases.GetUserListTest">
    <failure type="org.json.JSONException" message="Value {&amp;quot;path&amp;quot;:&amp;quot;\/v1\/getUserList&amp;quot;,&amp;quot;error&amp;quot;:&amp;quot;Not Found&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;No message available&amp;quot;,&amp;quot;timestamp&amp;quot;:&amp;quot;2019-02-14T09:21:18.871+0000&amp;quot;,&amp;quot;status&amp;quot;:404} of type org.json.JSONObject cannot be converted to JSONArray">
      <![CDATA[org.json.JSONException: Value {"path":"\/v1\/getUserList","error":"Not Found","message":"No message available","timestamp":"2019-02-14T09:21:18.871+0000","status":404} of type org.json.JSONObject cannot be converted to JSONArray
at org.json.JSON.typeMismatch(JSON.java:111)
at org.json.JSONArray.<init>(JSONArray.java:96)
at org.json.JSONArray.<init>(JSONArray.java:108)
at com.course.cases.GetUserListTest.getJsonResult(GetUserListTest.java:66)
at com.course.cases.GetUserListTest.getuserlist(GetUserListTest.java:31)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- getuserlist -->
</testsuite> <!-- 用户管理系统测试用例 -->
